dbt/include/snowflake/macros/adapters.sql

```diff
@@ -134,20 +134,29 @@
 
 {% endmacro %}
 
-{% macro snowflake__list_relations_without_caching(schema_relation, max_iter=10, max_results_per_iter=10000) %}
+{% macro snowflake__list_relations_without_caching(schema_relation, max_iter=10, max_results_per_iter=10000, query_pre_hook=None) %}
```

Comment:
I know this is QUITE the move to make but I couldn't think of another way to test for when this flag is set for a test. I think this will come in handy when we do other `QUOTED_IDENTIFIERS_IGNORE_CASE`-related matters.

I stand by this as the best move I can make but I recognize it may not be enough per se.

Reply:
We found a much better way forward by rethinking the underlying logic for metadata columns :)